app/Console/Kernel.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Console;
+
+use Illuminate\Console\Scheduling\Schedule;
+use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
+
+class Kernel extends ConsoleKernel
+{
+    /**
+     * The Artisan commands provided by your application.
+     *
+     * @var array
+     */
+    protected $commands = [
+        //
+    ];
+
+    /**
+     * Define the application's command schedule.
+     *
+     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
+     * @return void
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Tag @return cannot be grouped with parameter tags in a doc comment](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)